vscode-wpilib/package.json

```diff
@@ -534,75 +521,67 @@
     "repository": "https://github.com/wpilibsuite/vscode-wpilib",
     "homepage": "https://github.com/wpilibsuite/vscode-wpilib",
     "scripts": {
-        "vscode:prepublish": "gulp build && npm run lint && webpack --mode production",
+        "vscode:prepublish": "gulp build && eslint . && webpack --config-node-env production",
         "clean": "rmdir-cli out",
         "lint": "eslint .",
-        "webpack": "webpack --mode development",
-        "webpack-watch": "webpack --mode development --watch",
+        "webpack": "webpack --config-node-env production",
+        "webpack-watch": "webpack --config-node-env development --watch",
         "unittest": "mocha -r ts-node/register -u tdd src/unittest/*.ts",
         "gulp": "gulp",
-        "test": "npm run webpack && node ./node_modules/vscode/bin/test",
+        "test": "webpack --config-node-env production && node ./node_modules/vscode/bin/test",
         "updateActivation": "gulp update-activation",
-        "vscePackage": "vsce package"
+        "vscePackage": "vsce package --no-dependencies"
     },
     "devDependencies": {
-        "@types/glob": "^7.1.3",
-        "@types/micromatch": "^4.0.1",
-        "@types/mkdirp": "^1.0.1",
-        "@types/mocha": "^8.0.3",
-        "@types/ncp": "^2.0.4",
-        "@types/node": "^10.2.0",
-        "@types/node-fetch": "^2.5.7",
-        "@types/triple-beam": "^1.3.1",
+        "@types/glob": "^7.2.0",
+        "@types/micromatch": "^4.0.9",
+        "@types/mkdirp": "^1.0.2",
+        "@types/mocha": "^10.0.10",
+        "@types/ncp": "^2.0.8",
+        "@types/node": "^20.19.0",
```

Comment:
If we're bumping the `@types/node` version, the VSCode target should be bumped too. Looks like Node 20 [first shipped in Electron 29](https://www.electronjs.org/docs/latest/tutorial/electron-timelines), which was shipped [in VSCode 1.90](https://code.visualstudio.com/updates/v1_90#_electron-29-update).